### PR TITLE
Add safeguarding document

### PR DESCRIPTION
**Pull request by gedankenstuecke**
Thanks @GeorgiaHCA for writing up the AutSPACEs safeguarding document, and thanks for @KirstieJane for all the amazing input on it! I've taken the `docx` form and turned it into a more readable Markdown version as we had discussed 🎉

### CODE REVIEW

File: 00-project-documentation/community/Safeguarding.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+# Safeguarding risks & mitigation strategies
+
+Collectively, the AutSPACEs team together with the larger community have identified a number of potential risks that participating in different AutSPACEs events that could particularly affect vulnerable adults (e.g. autistic and co-occuring conditions). Jointly we have developed a wide range of mitigation strategies to minimize those risks. 
+
+The team of AutSPACEs event organisers and faciliators (in particular Sophia Batchelor, Georgia Aitkenhead & Bastian Greshake Tzovaras) are responsible for the implementation of these actions.
```
Comment (llewelld) [inline 00-project-documentation/community/Safeguarding.md:5]:
```suggestion
The team of AutSPACEs event organisers and facilitators (in particular Sophia Batchelor, Georgia Aitkenhead & Bastian Greshake Tzovaras) are responsible for the implementation of these actions.
```
Minor typo.

File: 00-project-documentation/community/Safeguarding.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+| C | We make it clear participants are free to leave at any time without giving a justification, and that they do not have to respond to any questions they do not wish to. | R2, R3, R5 |
+| D | Have multiple facilitators available to ensure all participants’ needs are met – such as providing them with a quiet space to decompress, or access to the bathroom. | R3 |
+| E | Provide all attendees with a schedule in advance including frequent breaks. | R3 |
+| F | Give clear directions and a description of the working space. | R3|
+| G | Hold sessions outside of rush hour. | R3 |
+| H | Keep to schedule as closely as possible. | R3 |
+| I | Ask group what support or adjustments they need to feel comfortable, such as dimmed lights, and follow their suggestions where possible. | R3|
+| J | Check in frequently to make sure participants feel comfortable and their needs are met. |R3, R5| 
+| K | Have autistic co-researchers actively involved in setting agendas and priorities to ensure alignment of goals with both participant needs and EDI strategy of the Institute | R4 |
```
Comment (llewelld) [inline 00-project-documentation/community/Safeguarding.md:42]:
```suggestion
| K | Have autistic co-researchers actively involved in setting agendas and priorities to ensure alignment of goals with both participant needs and the EDI strategy of the Institute | R4 |
```
Small suggestion.

**Review by llewelld — APPROVED** [review]
I'm not in a position to judge the content (which at any rate looks excellent to my untrained eye). I made some minor typo-fix sugggestions.

One other very minor point is that there seems to be inconsistent use of fullstops at the end of each item.

Very small suggestions; otherwise LGTM so approving.

File: 00-project-documentation/community/Safeguarding.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
+# Safeguarding risks & mitigation strategies
+
+Collectively, the AutSPACEs team together with the larger community have identified a number of potential risks that participating in different AutSPACEs events that could particularly affect vulnerable adults (e.g. autistic and co-occuring conditions). Jointly we have developed a wide range of mitigation strategies to minimize those risks.
```
Comment (llewelld) [inline 00-project-documentation/community/Safeguarding.md:3]:
```suggestion
Collectively, the AutSPACEs team together with the larger community have identified a number of potential risks that participating in different AutSPACEs events that could particularly affect vulnerable adults (e.g. autistic and co-occurring conditions). Jointly we have developed a wide range of mitigation strategies to minimize those risks. 
```
Small typo.

File: 00-project-documentation/community/Safeguarding.md (hunk 1 above)
Comment (gedankenstuecke) [reply 00-project-documentation/community/Safeguarding.md:5]: It shows which parts I immediately wrote on GH and that aren't just copy & paste from the Turing's docx form 😂

**Pull request merged**